kotlinx-coroutines-test/build.gradle.kts

```diff
@@ -2,6 +2,24 @@
  * Copyright 2016-2021 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
  */
 
-dependencies {
-    implementation(project(":kotlinx-coroutines-debug"))
+val experimentalAnnotations = listOf(
+    "kotlin.Experimental",
+    "kotlinx.coroutines.ExperimentalCoroutinesApi",
+    "kotlinx.coroutines.InternalCoroutinesApi"
+)
+
+kotlin {
```

Comment:
I guess it's time to actually maintain (instead of duct-taping) our build scripts.
This part is the copy-paste from core-module, please introduce .kts with common utility